packages/core/src/filters/Filter.ts

```diff
@@ -230,7 +252,14 @@ export class Filter extends Shader
     /** The WebGL state the filter requires to render. */
     state: State;
 
-    protected _resolution: number;
+    /**
+     * The resolution / device pixel ratio of the canvas.
+     *
+     * This is set to automatically match the renderer resolution by default, but can be overridden by setting manually.
+     * @default PIXI.settings.RESOLUTION
+     */
```

Comment:
Could you mark these docs at `@private` so that they aren't included in the docs? These properties are only public for FilterSystem and users should not be changing them.

Reply:
You know what, there's a lot of leaked private properties in the docs. We'll do a clean-up later of these things.